### PR TITLE
Kills waldo

### PR DESCRIPTION
**Pull request by FalloutFalcon**
## About The Pull Request

Swaps out the waldo legion for a syndicate infiltrator
Also sorts the legion corpses and shrinks the code a little with some loops
also removes the elite suit from legion pool..

## Why It's Good For The Game

Waldo felt a little lrp and he had alot of spell books and magic items which we want less of

Also i died to a engineer cause he had a blind spell. Must remove from game

## Changelog

:cl:
tweak: Replaces waldo with infiltrator
balance: Removes the elite suit from the legion pool
/:cl:

### CODE REVIEW

**Comment by Sun-Soaked** [comment]
It's time to unfind him. Goodbye, waldo

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm [hunk 1]
```diff
@@ -909,91 +734,164 @@
...
+			gloves = /obj/item/clothing/gloves/color/latex/nitrile/infiltrator
+			suit = /obj/item/clothing/suit/armor/vest/infiltrator
+			head = /obj/item/clothing/head/helmet/infiltrator
+			shoes = /obj/item/clothing/shoes/combat/sneakboots
+			ears = /obj/item/radio/headset
+			if(prob(70))
+				glasses = pickweight(list(
+					/obj/item/clothing/glasses/regular/circle = 1,
+					/obj/item/clothing/glasses/sunglasses = 3,
+					/obj/item/clothing/glasses/thermal/syndi = 2,
```
Comment (meemofcourse) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm]: Giving them thermals is a bad idea, regardless of how rare they are.
```suggestion
					/obj/item/clothing/glasses/night = 2,
```

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm [hunk 2]
```diff
@@ -909,91 +734,164 @@
...
+					)
+				)
+
+		if("Infiltrator")//WE FINALLY FOUND HIM
+			uniform = /obj/item/clothing/under/syndicate/bloodred
+			gloves = /obj/item/clothing/gloves/color/latex/nitrile/infiltrator
+			suit = /obj/item/clothing/suit/armor/vest/infiltrator
+			head = /obj/item/clothing/head/helmet/infiltrator
+			shoes = /obj/item/clothing/shoes/combat/sneakboots
+			ears = /obj/item/radio/headset
```
Comment (meemofcourse) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm]:
```suggestion
			ears = /obj/item/radio/headset/syndicate
```

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm [hunk 3]
```diff
@@ -909,91 +734,164 @@
...
+				glasses = pickweight(list(
+					/obj/item/clothing/glasses/regular/circle = 1,
+					/obj/item/clothing/glasses/sunglasses = 3,
+					/obj/item/clothing/glasses/thermal/syndi = 2,
+					/obj/item/clothing/glasses/night = 2
+					)
+				)
+			mask = /obj/item/clothing/mask/infiltrator
+			id = /obj/item/card/id/syndicate
+			back = /obj/item/storage/backpack/satchel/leather
```
Comment (meemofcourse) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm]: Perhaps give them a normal or a sec backpack.

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm [hunk 4]
```diff
@@ -909,91 +734,164 @@
...
+				mask = /obj/item/clothing/mask/gas/sechailer/swat
+				id = /obj/item/card/id/solgov
+			if(prob(85))
+				back = /obj/item/storage/backpack
+				backpack_contents = list()
+				for(var/i = 1 to 3)
+					if(prob(75))
+						backpack_contents += pickweight(list(
+							/obj/item/reagent_containers/hypospray/medipen/stimpack/traitor = 1,
+							/obj/item/storage/firstaid/tactical = 1,
```
Comment (meemofcourse) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm]: This has stimulants, medical night vision, and a combat defib in it. Change it to a conventional or medical kit, please.

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm [hunk 5]
```diff
@@ -909,91 +734,164 @@
...
+							/obj/item/gun/ballistic/automatic/pistol/solgov = 1,
+							/obj/item/gps = 1,
+							/obj/item/stock_parts/cell/gun/upgraded = 2,
+							/obj/item/ammo_box/magazine/pistol556mm = 3,
+							/obj/item/desk_flag/solgov = 3,
+							/obj/item/stack/marker_beacon/ten = 3,
+							/obj/item/detective_scanner = 2,
```
Comment (meemofcourse) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm]: This det scanner is pointless.

**Review by meemofcourse — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm (hunk 4 above)
Comment (FalloutFalcon) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm]: I didnt add this but i do agree its 2 powerful

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm (hunk 1 above)
Comment (FalloutFalcon) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm]: We give thermals in the komodo and two ruins, i wouldn't say its too powerful?

**Comment by axelzonvolt** [comment]
flashdark did nothing wrong

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm [hunk 6]
```diff
@@ -558,63 +585,26 @@
...
-						/obj/item/stack/sheet/animalhide/goliath_hide = 3,
-						/obj/item/hivelordstabilizer = 2,
-						/obj/item/stack/marker_beacon/ten = 2,
-						/obj/item/mining_scanner = 2,
-						/obj/item/extinguisher/mini = 2,
-						/obj/item/kitchen/knife/combat/survival = 3,
-						/obj/item/flashlight/seclite=3,
-						/obj/item/stack/sheet/sinew = 3,
-						/obj/item/stack/sheet/bone = 3
+				for(var/i = 1 to 3)
```
Comment (MarkSuckerberg) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm]:
```suggestion
				for(var/count in 1 to 3)
```

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm [hunk 7]
```diff
@@ -811,60 +671,25 @@
...
-						/obj/item/extraction_pack = 3,
-						/obj/item/stack/sheet/mineral/plasma/twenty = 3,
-						/obj/item/stack/marker_beacon/ten = 3,
-						/obj/item/mining_scanner = 2,
-						/obj/item/extinguisher/mini = 3,
-						/obj/item/flashlight/seclite=3,
-						/obj/item/research_notes/loot/medium = 3,
-						/obj/item/stack/sheet/metal/fifty = 3,
-						/obj/item/research_notes/loot/big = 1
+				for(var/i = 1 to 3)
```
Comment (MarkSuckerberg) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm]:
```suggestion
				for(var/count in 1 to 3)
```

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm [hunk 8]
```diff
@@ -411,14 +411,14 @@
...
+		"SRM" = 10,
 		"Oldminer" = 8,
-		"Kobold" = 5,
+		"Ashwalker" = 7,
 		"Golem" = 6,
-		"SRM" = 10,
-		pick("Shadow", "YeOlde", "Operative", "Cultist") = 4
+		"Kobold" = 5,
+		"Soldier" = 3,
+		"Infiltrator" = 3,
```
Comment (MarkSuckerberg) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm]: honestly I think it needs to be lower than even this, that's quite a lot of good syndie gear honestly

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]

**Comment by FalloutFalcon** [comment]
Slowly disliking the current legion spawn rates. Might restart this pr as a more in depth rework of the corpses

**Pull request closed without merging**